tests/alfacase/test_generate_case_description_docstring.py

```diff
@@ -13,8 +13,8 @@
     case_description.CaseDescription
 )
 
-# Useful for debugging
-# ALL_CLASSES_THAT_NEEDS_SCHEMA = [case_description.PvtModelsDescription]
+# # Useful for debugging
+# ALL_CLASSES_THAT_NEEDS_SCHEMA = [case_description.TrendOutputDescription]
```

Comment:
I don't want to block the PR merging for this.
Next time maybe do not commit this change =)